doc/doxy/Doxyfile

```diff
@@ -101,6 +101,7 @@ ALIASES                = qbk{1}="\xmlonly <qbk>\1</qbk> \endxmlonly" \
                          details_calc{1}="The free function \1 calculates the \1 of a geometry" \
                          details_calc2{2}="The free function \1 calculates the \2 of two geometries" \
                          details_calc2{1}="The free function \1 calculates the \1 of two geometries" \
+                         details_free_function{3}="The free function [^\1] calculates the \2 \3 two geometries" \
```

Comment:
The `\3` is for `between`. We can harmonize the macros later but I didn't want to make this PR too large.